docs/source/_developers/primer.md

```diff
@@ -0,0 +1,234 @@
+
+:::{tip}
+Set *Soft-Wrap* to be set to 100 characters by default when viewing `*.md`
+
+<figure>
+  <img src="../_static/image/settings-softwrap.png" alt="Soft-Wrap Setting" style="width:100%">
+  <figcaption>Fig. Soft-Wrap set to 100 characters in PyCharm's <i>General</i> settings.</figcaption>
+</figure>
+:::
+
+:::{note}
+Add line-breaks before and after the opening and closing of directives (i.e., outside the struct). Only add a single line-break after opening, which is the case for most directives; for those that do not require any inputs but titles (e.g., `:::{directive} Title`), omit line-breaks after the opening.
+:::
+::::
+
+::::{important}
+These nested admonitions were done, mostly, to demonstrate how it can be done. There are exceptions, but this is quite nested: be sure to consider if the flow of information is done most effectively when nests deeper, or even as as deep as this example (i.e., five colons `:::::` deep).
+::::
+:::::
+
+Consistent formatting makes your documentation look professional and easy to read. Use consistent formatting for headings, lists, tables, and other elements, and adequately nest subheadings and lists. Here are some guidelines:
+
+- **Headings**: Use headings and subheadings to structure your content. In MyST, use # for level 1 headings, ## for level 2, and so on.
+- **Lists:** Use bulleted lists for unordered items and numbered lists for ordered items. In MyST, use or  for unordered lists and 1. for ordered lists.
+- **Bold and Italics**: Use bold to highlight important concepts and italics to indicate new terms. In MyST, use bold text** for bold and italic text* for italics.
+- **Code Blocks**: Use code blocks to present code snippets. In MyST, use triple backticks to create code blocks and specify the language for syntax highlighting.
+- **Admonitions**: Use MyST syntax to create admonitions highlighting notes, tips, warnings, and other important information. For example, use `:::{note}` or `:::{warning}` to create admonitions.
+
+##  Code Example
+Code examples can help readers understand how to use a feature or solve a problem. Here are some guidelines to follow:
+- **Clarity**: Ensure that your code is easy to understand. Include comments to explain complex or essential parts of the code.
+- **Syntax** Highlighting: Use syntax highlighting to make your code easier to read. In MyST, you can specify the language for syntax highlighting in code blocks (see [Myst Source code and APIs]).
+- **Error Handling**: If relevant, show how to handle errors and exceptions.
+- **Consistency**: Ensure your code examples follow the same coding style, including indentation and naming conventions.
+- **GitHub**: Reference the repo if snippets from another RC GitHub project is referred.
+
```

Comment:
It might be worth pointing out if you have several code snips to focus it into one code block with comments for clean rendering on the webpage. 

To use \ to put breaks in for long code lines to you don't have to scroll for the entire code statement.

Reply:
All set. Checkout Note and Tip added in Code section.